pkg/clusteragent/clusterchecks/dispatcher_main.go

```diff
@@ -196,6 +208,22 @@ func (d *dispatcher) reset() {
 	d.store.reset()
 }
 
+// scanExtendedDanglingConfigs scans the store for extended dangling configs
+// The attemptLimit is the number of times a reschedule is attempted before
```

Comment:
where is attemptLimit set?

Reply:
Sorry this was an outdated comment. Missed updating it to the new impl. Fixed now!